rosidl_generator_cpp/package.xml

```diff
@@ -15,6 +15,8 @@
   <!-- This is needed for the rosidl_message_type_support_t struct and visibility macros -->
   <build_export_depend>rosidl_generator_c</build_export_depend>
 
+  <build_depend>rosidl_generator_c</build_depend>
```

Comment:
Please move the build dependency above the build export dependencies (around line 11).

Reply:
Will do.